README.md

```diff
@@ -1,35 +1,51 @@
 [![run with docker](https://img.shields.io/badge/run%20with-docker-0db7ed?labelColor=000000&logo=docker)](https://www.docker.com/)
 [![run with singularity](https://img.shields.io/badge/run%20with-singularity-1d355c.svg?labelColor=000000)](https://sylabs.io/docs/)
 [![Launch on Nextflow Tower](https://img.shields.io/badge/Launch%20%F0%9F%9A%80-Nextflow%20Tower-%234256e7)](https://tower.nf/launch?pipeline=https://github.com/Arcadia-Science/hifi2genome)
 
-**Arcadia-Science/hifi2genome** is a workflow for assembling PacBio Hifi reads into single genomes and performing QC stats on the assembled genomes.
+**Arcadia-Science/hifi2genome** is a workflow for assembling PacBio Hifi reads and performing QC stats on the resulting assemblies.
 
 The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool to run tasks across multiple compute infrastructures in a very portable manner. It uses Docker/Singularity containers making installation trivial and results highly reproducible. The [Nextflow DSL2](https://www.nextflow.io/docs/latest/dsl2.html) implementation of this pipeline uses one container per process which makes it much easier to maintain and update software dependencies. Where possible, these processes have been submitted to and installed from [nf-core/modules](https://github.com/nf-core/modules) in order to make them available to all nf-core pipelines, and to everyone within the Nextflow community!
 
 ## Pipeline summary
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps in the pipeline -->
+This workflow only supports PacBio Hifi reads in `.fastq.` format for producing an assembly and QC stats. Each set of reads are assembled with `flye`, assembly statistics summarized with `QUAST`, reads mapped back to the assembly with `minimap2` and lineage specific QC stats produced with `BUSCO`. The workflow then reports the QC stats from `QUAST` and `BUSCO` into an `.html` report with `MultiQC`.
 
 ## Quick start
 
-<!-- TODO: Fill in short bullet-pointed list of the default steps to get the pipeline up and running -->
+1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=22.10.1`)
+
+2. Install [`Docker`](https://docs.docker.com/engine/installation/), [`Singularity`](https://www.sylabs.io/guides/3.0/user-guide/) (you can follow [this tutorial](https://singularity-tutorial.github.io/01-installation/)), or [`Conda`](https://conda.io/miniconda.html). You can use conda to install Nextflow itself but use it for managing software within pipelines as a last resort. We recommend using `Docker` if possible as this has been tested most frequently. See the nf-core [docs](https://nf-co.re/usage/configuration#basic-configuration-profiles)) for more information.
+
+3. Test the pipeline on a minimal test dataset with a single command:
+
+   ```console
+   nextflow run Arcadia-Science/hifi2genome -profile test,YOURPROFILE --outdir <OUTDIR>
```

Comment:
This has a default samplesheet input yeah?

Reply:
yes it is declared in the test profile which is why you don't have to specify `--input`